__tests__/articles.test.js

```diff
@@ -51,3 +52,46 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("/api/articles", () => {
+  test("200 - GET - will return an array of all article objects with the of author, title, article_id, topic, created_at, votes and article_img_url, in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const articles = res.body.articles;
+        expect(articles).toHaveLength(13);
+        expect(Array.isArray(articles)).toBe(true);
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).not.toHaveProperty("body");
+        });
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("200 - GET - will return an array of all article objects with a comment_count property. This will be the total count of comments with this article_id.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const articles = res.body.articles;
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("comment_count", expect.any(Number));
+        });
+      });
+  });
+  test("404 - GET - sends an appropriate status and error message when an invalid URL is passed", () => {
```

Comment:
You already have a test for this, this 404 is not related to this end point as it is specifically for just missing urls. This does not need to be here. You only need to test for this once and in it's own describe block.

Reply:
<img width="987" alt="Screenshot 2024-08-26 at 13 21 00" src="https://github.com/user-attachments/assets/5d75f547-f444-4bc3-83f7-5aa546176581">